packages/expandable/index.js

```diff
@@ -65,6 +61,8 @@ class WarpExpandable extends kebabCaseAttributes(WarpElement) {
       ::slotted(:last-child) {
         margin-bottom: 0px !important;
       }
+
+      @unocss-placeholder;
```

Comment:
It could be this is fine, but I thought that we should keep all classes used in our components in the [@warp-ds/css/component-classes/index.js](https://github.com/warp-ds/css/blob/next/component-classes/index.js), so that  all necessary CSS is available through WarpElements styles - in this case through `components.css` stylesheet. 
Have you checked by linking this repo if this component can be used without issues in a project that is on old @warp-ds/uno version?